### ISSUE TITLE
Write docstrings for the inference operators

### ISSUE THREAD

**Issue by karlhigley** [NVIDIA-Merlin/systems#3]
- [ ] Feast
- [ ] Tensorflow
- [ ] Faiss
- [ ] Filtering
- [ ] Unroll
- [ ] Softmax
